src/test/java/seedu/address/logic/commands/SortStrengthCommandTest.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getGenericAddressBookWithSortedStrengths;
+import static seedu.address.testutil.TypicalPersons.getGenericAddressBookWithUnsortedStrengths;
+import static seedu.address.testutil.TypicalTasks.getTypicalTaskBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.StrategyBoard;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for SortStrengthCommand.
+ */
+public class SortStrengthCommandTest {
+
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
```

Comment:
Could be renamed to something that fits the common style of X_Y_Z.

Reply:
The rationale for this name is that it is not a test method, but rather an initialization method (hence the `@BeforeEach` annotation).